src/chart/pie/PieSeries.ts

```diff
@@ -299,6 +318,11 @@ class PieSeriesModel extends SeriesModel<PieSeriesOption> {
             opacity: 1
         },
 
+        showBackground: false,
+        backgroundStyle: {
+            opacity: 1
```

Comment:
Setting `opacity: 1` by default is confusing and unconvenient. How about make it 0.25 by default?